index.html

```diff
@@ -1499,7 +1499,7 @@ <h2>Test suite</h2>
         The W3C Verifiable Claims Working Group has produced a 
         <a href="https://github.com/w3c/vc-test-suite/">test suite</a> in order 
         for implementers to validate their conformance with the current specifications. 
-        Any conforming implementation MUST pass all tests in the test suite.
+        Conforming implementation are encouraged to pass all tests in the test suite.
```

Comment:
`Conforming implementation are`
-> `Conforming implementations are`

Reply:
Grrr...it's late. Fixed!